src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -41,6 +43,17 @@ public void execute_personAcceptedByModel_addSuccessful() throws Exception {
         assertEquals(Arrays.asList(validPerson), modelStub.personsAdded);
     }
 
+    @Test
+    public void execute_patientAcceptedByModel_addSuccessful() throws Exception {
```

Comment:
Regarding AgeTest.java, how do you suggest I generate values based on the constants? These examples were chosen because of which numbers they start from [eg starting with "1", starting with "2"].